cmd/main.go

```diff
@@ -39,7 +39,10 @@ func main() {
 	flag.StringVar(&options.Namespace, "namespace", "", "only migrate PVCs within this namespace")
 	flag.BoolVar(&options.SetDefaults, "set-defaults", false, "change default storage class from source to dest")
 	flag.BoolVar(&options.VerboseCopy, "verbose-copy", false, "show output from the rsync command used to copy data between PVCs")
+	flag.BoolVar(&options.PreSyncMode, "pre-sync-mode", false, "for RWX PV only: create the new PVC and copy the data, but do not swap the PVC yet")
```

Comment:
this actually shouldn't be RWX-exclusive - RWO volumes also allow multiple pods to mount the same PVC at once as long as they're located on the same instance

Reply:
good point.
<img width="783" alt="image" src="https://github.com/user-attachments/assets/4bd22388-1187-4211-8ccf-e443be3475c8">

however when I tried it it didn't work, maybe because the pvmigrate pod had been scheduled on another node 🤔 

I will run a few more tests and see how this goes.